google/cloud/bigquery_storage_v1/reader.py

```diff
@@ -0,0 +1,608 @@
+# Copyright 2020 Google LLC
```

Comment:
Are we doing anything to keep this in sync with v1beta1? Fine if no, but if yes let's add a DO NOT EDIT comment near the top.

Reply:
I'd consider keeping them sync if there weren't significant changes in the requests/responses, but with the differences I think it's easier to maintain them independently.